skills_DS/base/templates/base/register.html

```diff
@@ -1,17 +1,22 @@
-{% extends 'base/base.html' %} 
-{% load crispy_forms_tags %} 
-{% block content %}
+{% extends 'base/base.html' %} {% load crispy_forms_tags %} {% block content %}
 <div class="container content-section">
-    <form method="POST">
-      {% csrf_token %}
-      <fieldset class="form-group">
-        <h2>Register</h2>
-        {{ form|crispy }}
-      </fieldset>
-      <div class="form-group">
-        <button class="btn btn-primary" type="submit">Register</button>
-      </div>
-    </form>
-    <small class="text-muted">Already have an account? <a href="{% url 'login'%}">sign in</a></small>
+  <form method="POST">
+    {% csrf_token %}
+    <fieldset class="form-group">
+      <h2>Register</h2>
+      {{ form|crispy }}
+    </fieldset>
+    <textarea style="height: 400px; width: 400px; overflow-y: scroll" onscroll="onScroll()" id="consentForm" class="form-control">
```

Comment:
Should probably add `readonly` attribute to prevent users from editing the texts

Reply:
ah, good catch